playbooks/gce/openshift-node/config.yml

```diff
@@ -12,6 +12,11 @@
   hosts: "tag_env-host-type-{{ oo_env }}-openshift-master"
   connection: ssh
   user: root
+  gather_facts: yes
+  tasks:
+    - command: 'curl "http://metadata.google.internal/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip" -H "Metadata-Flavor: Google"'
+      register: output
+    - set_fact: gce_public_ip="{{ output.stdout }}"
```

Comment:
Odd, it should always be there when pulled from dynamic inventory...  Otherwise, one of the aws modules would need to be run against the host to generate those facts.